gRPC-Core.podspec

```diff
@@ -99,6 +99,85 @@ Pod::Spec.new do |s|
+                              'third_party/libuv/include/uv/threadpool.h',
+                              'third_party/libuv/include/uv/tree.h',
+                              'third_party/libuv/include/uv/unix.h',
+                              'third_party/libuv/include/uv/version.h'
+  end
+
+  s.subspec 'Libuv-Implementation' do |ss|
+    ss.header_dir = ''
+    ss.header_mappings_dir = 'third_party/libuv/src'
+    ss.dependency "#{s.name}/Libuv-Interface", version
+    ss.libraries = 'dl'
+    ss.compiler_flags = '-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -pthread --std=gnu89 -pedantic -O2 -Wno-error'
+
+    ss.source_files = 'third_party/libuv/src/fs-poll.c',
+                      'third_party/libuv/src/idna.c',
+                      'third_party/libuv/src/inet.c',
+                      'third_party/libuv/src/random.c',
+                      'third_party/libuv/src/strscpy.c',
+                      'third_party/libuv/src/threadpool.c',
+                      'third_party/libuv/src/timer.c',
+                      'third_party/libuv/src/unix/async.c',
+                      'third_party/libuv/src/unix/bsd-ifaddrs.c',
+                      'third_party/libuv/src/unix/core.c',
+                      'third_party/libuv/src/unix/darwin-proctitle.c',
+                      'third_party/libuv/src/unix/darwin.c',
+                      'third_party/libuv/src/unix/dl.c',
+                      'third_party/libuv/src/unix/fs.c',
+                      'third_party/libuv/src/unix/fsevents.c',
+                      'third_party/libuv/src/unix/getaddrinfo.c',
+                      'third_party/libuv/src/unix/getnameinfo.c',
+                      'third_party/libuv/src/unix/kqueue.c',
+                      'third_party/libuv/src/unix/loop-watcher.c',
+                      'third_party/libuv/src/unix/loop.c',
+                      'third_party/libuv/src/unix/pipe.c',
+                      'third_party/libuv/src/unix/poll.c',
+                      'third_party/libuv/src/unix/process.c',
+                      'third_party/libuv/src/unix/proctitle.c',
+                      'third_party/libuv/src/unix/random-devurandom.c',
+                      'third_party/libuv/src/unix/random-getentropy.c',
+                      'third_party/libuv/src/unix/random-getrandom.c',
+                      'third_party/libuv/src/unix/signal.c',
+                      'third_party/libuv/src/unix/stream.c',
+                      'third_party/libuv/src/unix/tcp.c',
+                      'third_party/libuv/src/unix/thread.c',
+                      'third_party/libuv/src/unix/tty.c',
+                      'third_party/libuv/src/unix/udp.c',
+                      'third_party/libuv/src/uv-common.c',
+                      'third_party/libuv/src/uv-data-getter-setters.c',
+                      'third_party/libuv/src/version.c'
+    ss.private_header_files = 'third_party/libuv/include/uv.h',
```

Comment:
All files in `private_header_files` must also be in `source_files` section.

Reply:
Thanks for pointing this out! I refactor the `private_header_files` and `source_files` a little bit, and now all files in `private_header_files` are part of `source_files`. With these changes, tests are still passing.